src/implementations/vanilla/components/select/select.js

```diff
@@ -805,6 +805,15 @@ export class Select {
         this.search.focus();
         break;
 
+      case 'Tab':
+        e.preventDefault();
+        if (e.shiftKey) {
+          this.search.focus();
+        } else {
+          this.optionsContainer.querySelectorAll('input')[0].focus();
```

Comment:
i wonder..why are we taking over the default, which would be to move the focus to the first input available?
We want to handle the shit+tab, but we are not forced to prevent the default even when only tab is used

Reply:
that's true, but  with the default tab behaviour it will "tab" out of the multi select when reaching the last item. With the custom behaviour we have it should loops back to the top.
Now that I see this, this code will trigger an error if there are no options... I'll add a test